demo/lightning/image-segmentation/README.md

```diff
@@ -0,0 +1,167 @@
+            self.bucket_name = bucket_name
+
+            self.images_dataset = dataflux_mapstyle_dataset.DataFluxMapStyleDataset(
+                project_name=self.project_name,
+                bucket_name=self.bucket_name,
+                config=dataflux_mapstyle_dataset.Config(
+                    # This needs to be True to map images with labels
+                    sort_listing_results=True,
+                    prefix=kwargs["images_prefix"],
+                ),
+            )
+
+            self.labels_dataset = dataflux_mapstyle_dataset.DataFluxMapStyleDataset(
+                project_name=self.project_name,
+                bucket_name=self.bucket_name,
+                config=dataflux_mapstyle_dataset.Config(
+                    # This needs to be True to map images with labels
+                    sort_listing_results=True,
+                    prefix=kwargs["labels_prefix"],
+                ),
+            )
+    ```
+
+1. `data.py`
+
+    Implements the `Unet3DDataModule` class which inherits from PyTorch Lightning's `LightningDataModule`. An instance of the `DatafluxPytTrain` class is created in its `setup` function.
+
+    ```py
+    def setup(self, stage="fit"):
+        if stage == "fit":
+            train_data_kwargs = {
+                "patch_size": self.args.input_shape,
+                "oversampling": self.args.oversampling,
+                "seed": self.args.seed,
+                "images_prefix": self.args.images_prefix,
+                "labels_prefix": self.args.labels_prefix,
+                "transforms": get_train_transforms(),
+            }
+            self.train_dataset = DatafluxPytTrain(
+                project_name=self.args.gcp_project,
+                bucket_name=self.args.gcs_bucket,
+                **train_data_kwargs,
+            )
+            self.train_sampler = None
+            if self.args.num_workers > 1:
+                self.train_sampler = DistributedSampler(
+                    self.train_dataset,
+                    seed=self.args.seed,
+                    drop_last=True
+                )
+    
+    ```
+
+1. `train.py`
+
+    An instance of the `Unet3DDataModule` class is created and passed to the Pytorch Lightning `Trainer` instance's `fit` function in the `train_dataloaders` keyword argument.
+    ```py
+    model = Unet3DLightning(flags)
+    train_data_loader = Unet3DDataModule(flags)
+    trainer = pl.Trainer(
+        accelerator=flags.accelerator,
+        max_epochs=flags.epochs,
+        devices=flags.num_devices,
+        num_nodes=flags.num_nodes,
+        strategy="ddp",
+    )
+    trainer.fit(model=model, train_dataloaders=train_data_loader)
+
+    ```
+
+
+Please note that these demos are for educational and example purposes only, and have not been optimized for performance.
+
+
+## Setup
+1. Create a copy of the dataset in your GCP project
+    ```sh
+    gcloud storage cp -r gs://dataflux-demo-public/image-seg-dataset gs://{YOUR-GCS-BUCKET}
```

Comment:
Did I miss it? I'm not seeing the directory [here](https://pantheon.corp.google.com/storage/browser/dataflux-demo-public;tab=objects?forceOnBucketsSortingFiltering=true&e=-13802955&mods=monitoring_api_prod&project=dataflux-project&prefix=&forceOnObjectsSortingFiltering=false&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))). The command will also fail.

Reply:
I haven't created it yet. Couldn't find the bucket in our google.com project. Was going to reach out to you tomorrow :) I will copy the dataset over into it now. Thanks!

Reply:
Done!